### PR TITLE
Fix: print non PHP file processors changes in parallel runs

### PR DESCRIPTION
**Pull request by ian-zunderdorp**
Extra info:
- `PhpFileProcessor` already prints changed files itself.
- `NonPhpFileProcessor` does not do this yet, instead it relies on `ApplicationFileProcessor` which prints all changed files after applying all rectors
- `ApplicationFileProcessor` skips printing files when running in parallel
- This means that `NonPhpFileProcessor` only worked in non-parallel mode

The `RenameClassNonPhpRector` is the only one in rector-src that uses it (and this rector is used by rector-symfony), though there might be external rector packages that rely on this.

### CODE REVIEW

**Comment by ian-zunderdorp** [comment]
I couldn't find existing tests that covers (parallel) execution combined with actually writing files to the filesystem, so that's why there is only a src code change. The e2e tests only assert on reported output, which was already green.

**Review by samsonasik — APPROVED** [review]

**Comment by ian-zunderdorp** [comment]
Hi @TomasVotruba ,

What is the best way to go forward with this pull request? Is there any extra information you need, any changes?

File: src/NonPhpFile/NonPhpFileProcessor.php [hunk 1]
```diff
@@ -51,9 +55,12 @@ public function process(File $file, Configuration $configuration): array
             $file->changeFileContent($newFileContent);
         }
 
+        // Shouldn't this use `$file->hasChanged()`?
```
Comment (TomasVotruba) [inline src/NonPhpFile/NonPhpFileProcessor.php]: This should be removed, here we compare direct file contents.

Comment (ian-zunderdorp) [reply src/NonPhpFile/NonPhpFileProcessor.php]: :+1: , removed it.

**Comment by TomasVotruba** [comment]
Thank you :+1:

**Pull request merged**